sr_unix/dse_main.c

```diff
@@ -0,0 +1,218 @@
+/****************************************************************
```

Comment:
This is OK (no need to redo it) but it might have been easier on a reviewer if dse.c were *renamed* to dse_main and then a new dse.c created. I *think* (but am not sure) that this would have then shown a relevant diff for dse_main.c instead of showing the entire module as new. But again, not sure.

Reply:
Interesting. Rename is how I did dse_main and all the other *_main.c. Not sure what criteria git has for determining if a module is a rename vs a new one and why it was different only for dse_main.c